dockstore-webservice/src/main/java/io/dockstore/webservice/core/Category.java

```diff
@@ -0,0 +1,63 @@
+package io.dockstore.webservice.core;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.io.Serializable;
+import java.sql.Timestamp;
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.persistence.Transient;
+
+/**
+ * Describes a Category, which is a curated group of entries.  In this
+ * implementation, a Category is a Collection under the hood.
+ */
+public class Category implements Serializable {
+
+    @Transient
+    private Collection collection;
```

Comment:
Could also implement this by letting JPA handle it using inheritance
https://www.baeldung.com/hibernate-inheritance 

In other words, JPA will add and manage a type column in one of the strategies instead of you needing to use a "Special" status